flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/harness/HarnessTestBase.scala

```diff
@@ -491,13 +491,83 @@ class HarnessTestBase {
     distinctCountFuncName,
     distinctCountAggCode)
 
+  def createHarnessTester[KEY, IN, OUT](
+      dataStream: DataStream[_],
+      prefixOperatorName: String)
```

Comment:
How a bout add `aggFieldAlias: String = ""`, for resolve scenarios where GroupBy is included in multiple UNION clauses. e.g: 
```
      (SELECT b, max(a) as maxA FROM T GROUP BY b)
       UNION (
         SELECT b, min(a) as minA FROM (
          SELECT a, b FROM T GROUP BY a, b
         ) GROUP BY b
        )
      """.stripMargin
```
And we using this method as follows: `createHarnessTester(xx, "groupBy", "minA")`。
I didn't find a case where I had to test it in this way, it was just an enhanced suggestion.
What do you think?

Reply:
createHarnessTester will be used not only in agg related tests but also in other harness tests, such as stream join tests, temporal join tests, sort tests, etc. So field aggFieldAlias seems a little wired from my point of view. What about adding it when we really need it? At that time we may have a better idea on how such a field will look like. Thoughts?

Reply:
Hmm, If this tester will using in join test, we should add TwoInputTransformation check in the `extractExpectedTransformation` logic, then we also need add a xxName(may be not named aggFieldAlias), e.g.: `In multiple join scenes`？

Reply:
Yes, you're right. Will do that when updating the join related harness tests.